src/quad-tree.ts

```diff
@@ -184,22 +189,22 @@ export class QuadTree<T extends Rectangle> {
     // create new quad tree and set all fields to match this objects fields;
     const childQuad = new QuadTree<T>(
       Rectangle.from(rect),
-      structuredClone(capacity),
+      capacity,
       false
     );
 
-    childQuad.isLeaf = structuredClone(isLeaf);
+    childQuad.isLeaf = isLeaf;
```

Comment:
Ill keep this, but the reason I kept it originally is because adding a structurted clone modified the count, therefore cfhanging something, even though it really shouldn't have